angular-app/src/app/auth.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { AuthService } from './auth.service';
+
+describe('AuthService', () => {
```

Comment:
Is the build setup to run these tests automatically already?

Reply:
No, they're run with ng test using some external checker, but I haven't been very good about the test-driven development.